CMakeLists.txt

```diff
@@ -14,7 +15,7 @@ if (WIN32)
 endif()
 
 # Set the default install path to /usr
-if (NOT WIN32 AND CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+if (NOT APPLE AND NOT WIN32 AND CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
```

Comment:
Why the `NOT APPLE`? Are non-framework builds expected to install in /usr/local? There is no info about this on the commit's message.

Reply:
Installing directly into /usr/lib or /usr/bin on MacOS raises an error. I think it's due to System Integrity Protection.